src/components/Pages/Admin/View.vue

```diff
@@ -184,6 +185,10 @@ export default {
     },
 
     methods: {
+        groupColor(color) {
+            return(`background:rgb(${[...color]})`);
```

Comment:
OK, it turns out I was wrong about how format strings work, and you can just use `${color}` and it'll automatically omit the [ ] and just give you the comma separated items. (And you're actually doing it here, since `[...color]` is equivalent to `color`.)